shell.nix

```diff
@@ -4,7 +4,7 @@
   # the bky-as cli the example test works against is updated.
   #
   # This default value can be overwritten from the command line.
-  bkyAsVersion ? "e7a2c061da429c66dcfadaf6007e0a4161fea6dc",
+  bkyAsVersion ? "latest",
```

Comment:
I was wondering about keeping this always pinned to a specific version so the bky-as version does not change on devs underfoot and having the end to end tests, or scheduled tests in this repo, perform tests against "latest". Having it as "latest" does have the benefit of always catching delphi updates though before those scheduled tests are set up, so could be good to leave as "latest" in the meantime. What do you think?

Reply:
Not really sure.  But it was cool that @tahardi merged a PR into delphi while I was working on this. I created a new shell and everything just updated! It was kind of magical. :)

Reply:
My current vision is to have the SDK tests be pinned to a specific version of delphi the dev working on each PR knows it works with, and to have nightly tests check the sdk against delphi latest. I think since we don't have any nightly tests I'd be fine keeping "latest" here for now, or happy to chat more about env/test design for this repo later, whatever you end up going with.

Reply:
I do love that the env management with nix is working as hoped though!!

Reply:
for now, we are sticking with "latest" but may change it once we add in some nightly testing.